src/main/java/com/teragrep/pth10/steps/teragrep/bloomfilter/ToBloomFilter.java

```diff
@@ -46,127 +46,33 @@
 package com.teragrep.pth10.steps.teragrep.bloomfilter;
 
 import org.apache.spark.util.sketch.BloomFilter;
-import org.apache.spark.util.sketch.IncompatibleMergeException;
 
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
-import java.io.OutputStream;
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.List;
-import java.util.Objects;
 
 /**
  * BloomFilter from byte[] used in a constructor of TeragrepBloomFilter
  * 
  * @see TeragrepBloomFilter
  */
-public final class ToBloomFilter extends BloomFilter {
+public final class ToBloomFilter {
```

Comment:
naming has inverse thinking, ToBloomFilter is actually BloomFilterBlob which should have toBloomFilter as a method instead of fromBytes()